src/Export/XmlExport.php

```diff
@@ -130,16 +130,48 @@ private function exportSingleItem(
         return $xmlProduct->getXmlItem();
     }
 
-    private function isProductInCrossSellingCategory(ProductEntity $productEntity): bool
+    private function isProductInCrossSellingCategory(ProductEntity $productEntity): ?CategoryEntity
```

Comment:
If method name starts with `is` I would expect a boolean return, maybe `getEntityFromCrossSellingCategories`?